tests/YGDefaultValuesTest.cpp

```diff
@@ -25,9 +25,9 @@ TEST(YogaTest, assert_default_values) {
   ASSERT_EQ(YGPositionTypeRelative, YGNodeStyleGetPositionType(root));
   ASSERT_EQ(YGWrapNoWrap, YGNodeStyleGetFlexWrap(root));
   ASSERT_EQ(YGOverflowVisible, YGNodeStyleGetOverflow(root));
-  ASSERT_FLOAT_EQ(0, YGNodeStyleGetFlexGrow(root));
-  ASSERT_FLOAT_EQ(0, YGNodeStyleGetFlexShrink(root));
-  ASSERT_FALSE(YGNodeStyleGetFlexBasis(root).unit != YGUnitAuto);
+  ASSERT_TRUE(YGFloatIsUndefined(YGNodeStyleGetFlexGrow(root)));
```

Comment:
Currently the default is Undefined and we use that value inside ```YGResolveFlexGrow```. What do you think how we should use that with the ```flex``` shorthand? Should we use ```flex``` to set the ``flexGrow``` directly. But so we loose the previous set values.